File: java/Vehicle.java
public class Vehicle {
    private String numberPlate;
    private String brand;
    private int productionYear;
    private String color;

    // Constructor
    public Vehicle(String plate, String brand, String clr, int year) {
        this.numberPlate = plate;
        this.brand = brand;
        this.productionYear = year;
        this.color = clr;
    }

    // Getters
    public String getNumberPlate() {
        return numberPlate;
    }

    public String getBrand() {
        return brand;
    }

    public int getProductionYear() {
        return productionYear;
    }

    public String getColor() {
        return color;
    }

    // Setters
    public void setNumberPlate(String plate) {
        this.numberPlate = plate;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public void setProductionYear(int year) {
        this.productionYear = year;
    }

    public void setColor(String clr) {
        this.color = clr;
    }
}

File: php/Vehicle.php
<?php
class Vehicle {
    private $numberPlate;
    private $brand;
    private $productionYear;
    private $color;

    // Constructor
    public function __construct($plate, $brand, $clr, $year) {
        $this->numberPlate = $plate;
        $this->brand = $brand;
        $this->productionYear = $year;
        $this->color = $clr;
    }

    // Getters
    public function getNumberPlate() {
        return $this->numberPlate;
    }

    public function getBrand() {
        return $this->brand;
    }

    public function getProductionYear() {
        return $this->productionYear;
    }

    public function getColor() {
        return $this->color;
    }

    // Setters
    public function setNumberPlate($plate) {
        $this->numberPlate = $plate;
    }

    public function setBrand($brand) {
        $this->brand = $brand;
    }

    public function setProductionYear($year) {
        $this->productionYear = $year;
    }

    public function setColor($clr) {
        $this->color = $clr;
    }
}
